test/suites/queryservice/queryservice.conf.ts

```diff
@@ -0,0 +1,11 @@
+import { defaultTestSettings } from '../../setup/make-test-settings.js';
+import TestEnv from '../../setup/test-env.js';
+import wdioConfig from '../../setup/wdio.conf.js';
+
+export const testEnv = TestEnv.createWithDefaults( {
+	name: 'repo',
+	specs: [ 'specs/queryservice/*.ts' ],
+	composeFiles: [ ...defaultTestSettings.composeFiles ]
```

Comment:
Should default to exactly that if you leave out the `composeFiles` argument entirely